autoextract_spiders/spiders/autoextract_article.py

```diff
@@ -10,6 +10,17 @@ class ArticleAutoExtract(CrawlerSpider):
     name = 'articles'
     page_type = 'article'
 
+    frontera_settings = {
+        'HCF_PRODUCER_FRONTIER': 'autoextract',
+        'HCF_PRODUCER_SLOT_PREFIX': 'articles',
+        'HCF_PRODUCER_NUMBER_OF_SLOTS': 1,
+        'HCF_PRODUCER_BATCH_SIZE': 100,
```

Comment:
Have some doubts about good default values for produces/consumer batches, wdyt?

Reply:
I believe the values here are very much based on the volume. If we have thousands of URLs or more, per cycle (hourly maybe) than we definitely need more slots, because we want to have multiple producers writing to multiple slots and multiple consumers as well.
I didn't really find good default values, I think we should ask around :)

Reply:
Agree! At the same moment, I believe we do want to keep the default configuration as simple as possible, it also means having everything in the same spider (producer/consumer) for simplicity/performance reasons. And that basically means there should be a single slot in use by default, which enforces most of the settings here.

Reply:
Yes it can be like that initially and in the future we can run multiple instances of AutoExtract spiders, each instance having discovery + extraction, but using different seeds and writing to a different slot. Just an idea.